site/content/blog/2021-10-01-whats-new-in-svelte-october-2021.md

```diff
@@ -0,0 +1,97 @@
+---
+title: What's new in Svelte: October 2021
+description: A whole year of "What's new in Svelte"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Hey y'all 👋 It's been 1 year since "What's new in Svelte" started being cross-posted to the Svelte blog. I wanted to take this moment at the top to thank all of you for reading and for all the contributors every month. From the maintainers to everyone who posts their work in the Discord and Reddit, it's amazing to witness all the effort that goes into making the Svelte community great.
+
+Keep up the good work, everyone! Now, let's dive into this month's news...
+
+## New around Svelte
+
+- New additions to Svelte's export map now expose no-op versions of lifecycle functions for SSR. This prevents the need for bundlers to bundle external dependencies that imports `svelte` in order to resolve them to `svelte/ssr` (Svelte **3.43.0**)
```

Comment:
We had already export no-op versions since `3.39.0` under `svelte/ssr`. Though the change in `3.43.0` is the real way forward which exports the version under a `node` condition. Currently, `vite-plugin-svelte` is still using the old technique due to limitations in Vite and esbuild, so the second sentence is not quite true yet. I do want it to be true some day though.

To summarize, I think we should remove the second sentence.

```suggestion
- New additions to Svelte's export map now expose no-op versions of lifecycle functions for SSR (Svelte **3.43.0**)
```